[docs][skip ci] Maps read only access (#35561)
elastic/kibana: docs/maps/maps-getting-started.asciidoc

[role="screenshot"]
image::maps/images/sample_data_web_logs.png[]

[float]
[[maps-read-only-access]]
=== Read only access

> When you have insufficient privileges to create or save maps, the following 

Reviewer: ```suggestion
If you have insufficient privileges to create or save maps, a read-only icon 
```

> indicator in Kibana will be displayed. The buttons to create new maps or edit

Reviewer: ```suggestion
appears in the application header. The buttons to create new maps or edit
```
